<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>773</width>
    <height>604</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="src_comboBox">
       <item>
        <property name="text">
         <string>auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>en</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wyw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jp</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="src_textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>323</width>
         <height>557</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;你好&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn">
     <property name="text">
      <string>翻译</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="dst_comboBox">
       <item>
        <property name="text">
         <string>en</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wyw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jp</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="dst_textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>323</width>
         <height>557</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
